ProcessMaker/Models/EncryptedData.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace ProcessMaker\Models;
+
+use Illuminate\Support\Str;
+use ProcessMaker\Models\ProcessMakerModel;
+use ProcessMaker\Traits\HasUuids;
+
+class EncryptedData extends ProcessMakerModel
+{
+    use HasUuids;
+
+    const ENCRYPTION_METHOD = 'aes-256-cbc';
+
+    protected $connection = 'processmaker';
+
+    protected $table = 'encrypted_data';
+
+    /**
+     * The attributes that aren't mass assignable.
+     *
+     * @var array
+     */
+    protected $guarded = [
+        'id',
+        'uuid',
```

Comment:
Check this section, seems uuid must be returned

Reply:
now the endpoint only returns the uuid